Doc/reference/simple_stmts.rst

```diff
@@ -707,15 +707,14 @@ The :keyword:`import` statement
    keyword: from
 
 .. productionlist::
-   import_stmt: "import" `module` ["as" `name`] ( "," `module` ["as" `name`] )*
-              : | "from" `relative_module` "import" `identifier` ["as" `name`]
-              : ( "," `identifier` ["as" `name`] )*
-              : | "from" `relative_module` "import" "(" `identifier` ["as" `name`]
-              : ( "," `identifier` ["as" `name`] )* [","] ")"
+   import_stmt: "import" `module` ["as" `identifier`] ("," `module` ["as" `identifier`])*
```

Comment:
Why `name` was replaced by `identifier`?

Reply:
Because, when I added a link to name in future statements, it linked to the name in import statements. That's why I replaced name in both places to "identifier". Perhaps it can be solved in other way?

Reply:
Ah, I see, the problem was that `name` was defined in two places. Then using an `identifier` LGTM. In the Grammar file the same term is used for the names before and after 'as' in `import_from`.